index.html

```diff
@@ -208,16 +208,16 @@ <h1>Team</h1>
         <h1>Connect with Fossasia on Social Media</h1>
         <p>Be sure to follow us on social media!</p>
         <p class="large">
-          <a href="https://www.facebook.com/fossasia/" class="ti-facebook lgi" target="_blank" rel="noopener"></a>
-          <a href="https://twitter.com/fossasia" class="ti-twitter lgi" target="_blank" rel="noopener"></a>
-          <a href="https://www.instagram.com/fossasia/" target="_blank" rel="noopener" class="ti-instagram lgi"></a>
-          <a href="https://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ" target="_blank" rel="noopener" class="ti-youtube lgi"></a>
-          <a href="https://github.com/fossasia" target="_blank" class="ti-github lgi" rel="noopener"></a>
-          <a href="https://flickr.com/photos/fossasia" target="_blank" class="ti-flickr lgi" rel="noopener"></a>
-          <a href="https://www.linkedin.com/groups/FOSSASIA-3762811" target="_blank" class="ti-linkedin lgi" rel="noopener"></a>
-          <a href="https://plus.google.com/108920596016838318216" target="_blank" class="ti-google lgi" rel="noopener"></a>
-          <a href="https://codein.withgoogle.com/organizations/fossasia/" target="_blank" class="ti-settings lgi" rel="noopener"></a>
-          <a href="https://fossasia-slack.herokuapp.com/" target="_blank" class="fa fa-slack lgi" rel="noopener"></a>
+          <a href="https://fossasia-slack.herokuapp.com/" title="Slack" target="_blank" class="fa fa-slack lgi" style="display:initial"></a>
+          <a href="https://github.com/fossasia" title="GitHub" target="_blank" class="ti-github lgi"></a>
+          <a href="https://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ" title="YouTube" target="_blank" class="ti-youtube lgi"></a>
+          <a href="https://flickr.com/photos/fossasia" title="Flickr" target="_blank" class="ti-flickr lgi"></a>
+          <a href="https://www.facebook.com/fossasia/" title="Facebook" target="_blank" class="ti-facebook lgi"></a>
+          <a href="https://twitter.com/fossasia" title="Twitter" target="_blank" class="ti-twitter lgi"></a>
+          <a href="https://www.instagram.com/fossasia/" title="Instagram" target="_blank" class="ti-instagram lgi"></a>
+          <a href="https://www.linkedin.com/groups/FOSSASIA-3762811" title="LinkedIn" target="_blank" class="ti-linkedin lgi"></a>
+          <a href="https://plus.google.com/108920596016838318216" title="Google+" target="_blank" class="ti-google lgi"></a>
+          <a href="https://codein.withgoogle.com/organizations/fossasia/" title="GCI" target="_blank" class="ti-settings lgi"></a>
```

Comment:
You have deleted the rel="noopener", it opens the site in another thread to reduce performance loss. Please add it back